ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Extension.java

```diff
@@ -119,5 +120,6 @@ public void initializeParsers(ExtensionParsingContext context) {
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_6_0, EJB3Subsystem60Parser::new);
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_7_0, EJB3Subsystem70Parser::new);
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_8_0, EJB3Subsystem80Parser::new);
+        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, NAMESPACE_9_0, EJB3Subsystem90Parser::new);
```

Comment:
@Skyllarr Just noticed that it looks like there is also a `CURRENT_MODEL_VERSION` in this file that needs to be updated.

Reply:
@fjuma Fixed